src/scripts/Components/DocumentTitle.jsx

```diff
@@ -17,8 +18,19 @@ import { getMeta } from "@core/application";
  */
 export default class DocumentTitle extends React.Component {
     static propTypes = {
-        title: PropTypes.string,
         children: PropTypes.node,
+        classNames: PropTypes.string,
+        id: PropTypes.string,
+        title: PropTypes.string,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.classes = classNames(
```

Comment:
These classes should be determined in the top of the render function, otherwise changes to the classNames prop will not be respected.